demo-shell/src/app.config.json

```diff
@@ -739,6 +739,9 @@
     "name": "My Default Name",
     "processDefinitionName": "My default process def name"
   },
+  "adf-cloud-start-process": {
+    "name": "My Default Cloud Name"
```

Comment:
Does it support i18n?

Reply:
what do you mean? This is a custom name that you can add by default